### PR TITLE
Warn if pedometer missing & add button that opens manual

### PR DESCRIPTION
**Pull request by thevirtuoso1973**
In the unfortunate case where a marker/examiner tries out our app but does not have a step counter sensor on the device, this should help.

Also added a help button that opens the user manual as suggested by our TA,

@saachipahwa You should probably test out the help button on iOS.

https://user-images.githubusercontent.com/46009390/110532376-a1464c80-8114-11eb-9543-3dee6ce538bc.mp4

### CODE REVIEW

File: lib/pages/settings_page.dart [hunk 1]
```diff
@@ -83,7 +83,7 @@ class _SettingsPageState extends State<SettingsPage> {
               textAlign: TextAlign.center,
               style: Theme.of(context).textTheme.headline1),
           SizedBox(height: 10),
-          Expanded(flex: 2, child: settingsWidget),
+          Expanded(flex: 3, child: settingsWidget),
```
Comment (thevirtuoso1973) [inline lib/pages/settings_page.dart:86]: @saachipahwa Also I changed this cos the text was taking up too much space.

File: lib/pages/home_page.dart [hunk 2]
```diff
@@ -209,15 +230,45 @@ class _HomePageState extends State<HomePage> {
...
           mainAxisSize: MainAxisSize.min,
           children: [
-            heading,
-            Divider(),
+            Visibility(
+              visible: pedometerWarn == true,
+              child: warningBanner,
+            ),
             previousScoreHolder,
             Divider(),
```
Comment (saachipahwa) [inline lib/pages/home_page.dart:273]: This divider looks weird on iOS, do we need it?

File: lib/pages/home_page.dart [hunk 3]
```diff
@@ -209,15 +230,45 @@ class _HomePageState extends State<HomePage> {
...
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           mainAxisSize: MainAxisSize.min,
           children: [
-            heading,
-            Divider(),
+            Visibility(
+              visible: pedometerWarn == true,
+              child: warningBanner,
+            ),
             previousScoreHolder,
```
Comment (saachipahwa) [inline lib/pages/home_page.dart:272]: Could you add some padding around the previousScoreHolder?  Like Padding(
                padding: EdgeInsets.fromLTRB(0, 20, 0, 20),
                child: previousScoreHolder),

**Review by saachipahwa — APPROVED** [review]

**Comment by saachipahwa** [comment]
Help button works well!

File: lib/pages/settings_page.dart (hunk 1 above)
Comment (saachipahwa) [reply lib/pages/settings_page.dart:86]: No prob

**Comment by saachipahwa** [comment]
Just correct the nudge screen typo

**Review by saachipahwa — APPROVED** [review]
Looks good

**Pull request merged**